airbyte-webapp/src/locales/en.json

```diff
@@ -367,6 +367,14 @@
   "connection.catalogTree.sourceSchema": "'<source schema>",
   "connection.catalogTree.destinationSchema": "'<destination schema>",
 
+  "connection.geographyTitle": "Data residency",
+  "connection.geographyDescription": "Depending on your network configuration, you may need to <lnk>add IP addresses</lnk> to your allowlist.",
+  "connection.geography.auto": "Airbyte Default",
```

Comment:
I wonder if we want to put " (US)" at the end here, so be a bit more clear what our default is, specifically since it's likely not going to change any time soon.

Reply:
We discussed this and landed on "Airbyte default" without indicating of the underlying region. If we show "US" then users might falsely believe their data will always remain in the US. It also came up as feedback in the [demo UI](https://airbytehq-team.slack.com/archives/C046ZCUQTN0/p1666968303231229?thread_ts=1666966625.653199&cid=C046ZCUQTN0) I shared with the team.